@vaadin/vaadin-themable-mixin/register-styles errors in @vaadin/vaadin-lumo-styles/utilities

### Description

I'm using vaadin components @23.0.0-beta2 in my project with webpack5 and get the following error.
```sh
ERROR in ./node_modules/@vaadin/vaadin-lumo-styles/utilities/accessibility.js 6:0-68
Module not found: Error: Can't resolve '@vaadin/vaadin-themable-mixin/register-styles' in '/Users/naoak/frontend-web/node_modules/@vaadin/vaadin-lumo-styles/utilities'
Did you mean 'register-styles.js'?
BREAKING CHANGE: The request '@vaadin/vaadin-themable-mixin/register-styles' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```

Source files in [@vaadin/vaadin-lumo-styles/utilities](https://github.com/vaadin/web-components/tree/master/packages/vaadin-lumo-styles/utilities) should import `{ css }` from `@vaadin/vaadin-themable-mixin/register-styles.js` instead of from `@vaadin/vaadin-themable-mixin/register-styles`.

### Expected outcome

No errors.

### Actual outcome

Bundling errors happen.

### Environment

- v23.0.0-beta2 
- OSX 12.2.1

--- a/packages/vaadin-lumo-styles/utilities/accessibility.js
+++ b/packages/vaadin-lumo-styles/utilities/accessibility.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css } from '@vaadin/vaadin-themable-mixin/register-styles';
+import { css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
 
 export const accessibility = css`
   /* === Screen readers === */

--- a/packages/vaadin-lumo-styles/utilities/background.js
+++ b/packages/vaadin-lumo-styles/utilities/background.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css } from '@vaadin/vaadin-themable-mixin/register-styles';
+import { css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
 
 export const background = css`
   /* === Background color === */

--- a/packages/vaadin-lumo-styles/utilities/border.js
+++ b/packages/vaadin-lumo-styles/utilities/border.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css } from '@vaadin/vaadin-themable-mixin/register-styles';
+import { css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
 
 export const border = css`
   /* === Border === */

--- a/packages/vaadin-lumo-styles/utilities/flexbox-grid.js
+++ b/packages/vaadin-lumo-styles/utilities/flexbox-grid.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css } from '@vaadin/vaadin-themable-mixin/register-styles';
+import { css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
 
 export const flexboxAndGrid = css`
   /* === Align content === */

--- a/packages/vaadin-lumo-styles/utilities/layout.js
+++ b/packages/vaadin-lumo-styles/utilities/layout.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css } from '@vaadin/vaadin-themable-mixin/register-styles';
+import { css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
 
 export const layout = css`
   /* === Box sizing === */

--- a/packages/vaadin-lumo-styles/utilities/shadows.js
+++ b/packages/vaadin-lumo-styles/utilities/shadows.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css } from '@vaadin/vaadin-themable-mixin/register-styles';
+import { css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
 
 export const shadows = css`
   /* === Box shadows === */

--- a/packages/vaadin-lumo-styles/utilities/sizing.js
+++ b/packages/vaadin-lumo-styles/utilities/sizing.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css } from '@vaadin/vaadin-themable-mixin/register-styles';
+import { css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
 
 export const sizing = css`
   /* === Height === */

--- a/packages/vaadin-lumo-styles/utilities/spacing.js
+++ b/packages/vaadin-lumo-styles/utilities/spacing.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css } from '@vaadin/vaadin-themable-mixin/register-styles';
+import { css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
 
 export const spacing = css`
   /* === Margin === */

--- a/packages/vaadin-lumo-styles/utilities/typography.js
+++ b/packages/vaadin-lumo-styles/utilities/typography.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css } from '@vaadin/vaadin-themable-mixin/register-styles';
+import { css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
 
 export const typography = css`
   /* === Font size === */

--- a/packages/vaadin-lumo-styles/utility.js
+++ b/packages/vaadin-lumo-styles/utility.js
@@ -3,7 +3,7 @@
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/register-styles';
+import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
 import { accessibility } from './utilities/accessibility.js';
 import { background } from './utilities/background.js';
 import { border } from './utilities/border.js';

